flexmeasures/api/v3_0/assets.py

```diff
@@ -143,6 +145,16 @@ def post(self, asset_data: dict):
         asset = GenericAsset(**asset_data)
         db.session.add(asset)
         db.session.commit()
+
+        audit_log = AssetAuditLog(
```

Comment:
Maybe we can save a couple lines in this module (and/or others) by creating a util finction like this:

```
add_asset_audit_log(msg, asset_id)
```

I see you already made one for `write_audit_log_for_attribute_update()`. Maybe add them to the data model class or a (new) service module.